cmd/opentelemetry-collector/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM alpine:latest as certs
+RUN apk --update add ca-certificates
```

Comment:
Should this be same as collector Dockerfile? `RUN apk add --update --no-cache ca-certificates` with `--no-cache` option?

Reply:
+1, changed